ui/apps/platform/cypress/helpers/visit.js

```diff
@@ -130,3 +135,35 @@ export function visitWithStaticResponseForPermissions(
     waitForResponses(routeMatcherMapForAuthenticatedRoutes);
     waitForResponses(routeMatcherMap);
 }
+
+/**
+ * Visit page to test conditional rendering for central capabilities specified as response or fixture.
+ *
+ * { body: { resourceToAccess: { … } } }
```

Comment:
Copy-paste from permissions. Not a blocker to merge.

Is it `{ body: { … } }` correct? `body` consists of capabilities object?